samples/Foundatio.HostingSample/Jobs/Sample2Job.cs

```diff
@@ -22,7 +22,7 @@ public Sample2Job(ILoggerFactory loggerFactory)
 
     public Task<JobResult> RunAsync(CancellationToken cancellationToken = default)
     {
-        _lastRun = SystemClock.UtcNow;
+        _lastRun = DateTime.UtcNow;
```

Comment:
we should probably be using abstracted time as this is a sample and it does control the return result

Reply:
It's fine. Not all code needs to use virtualized time. If you aren't ever going to test it with fake time then it's pointless.